src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -129,4 +129,39 @@ public class VerticalAlignment : CustomGenericEnumerationDropDown
         public VerticalAlignment() : base("Vertical Alignment", typeof(Autodesk.Revit.DB.VerticalAlignmentStyle)) { }
     }
 
+    [NodeName("Fill Patterns")]
```

Comment:
I just found that I've deleted them in 2016 branch, but not in 2017. Can I just make one ore pull request to delete these 3 nodes from 2017 branch?